DataBase.py

```diff
@@ -0,0 +1,52 @@
+from peewee import *
+
+
+#DataBase connection
+conn = SqliteDatabase('data.db')
+
+class BaseModel(Model):
```

Comment:
There should be 2 blank lines before and after class/function definition (here and lower too)